### ISSUE TITLE
Remove selected state for section headers in nav drawer list 

### ISSUE THREAD

**Issue by Madeline-UX** [defenseunicorns/UnicornUI#220]
##Solution
Selected state should only be used on the element that is currently selected to make it clear at a glance what page the user is on. Having multiple list items selected makes the user have to think about which selected list item is the page they are currently on.

##Changes

- [ ] Use ul list with subheader
- [ ] Replace the collapsable subheader with non-collapsable subheader
- [ ] Only use selected state for page selected 
- [ ] subheaders are not selectable.

This issue is related to new design in #218 

## Reference images
 
current navigation drawer
<img width="281" alt="Screenshot 2023-05-16 at 10 40 46 AM" src="https://github.com/defenseunicorns/UnicornUI/assets/92826525/c2a500d4-f0fb-4142-8d78-2711ac70bdb6">

**Comment by TristanHoladay** [comment]
Currently on hold as dependent on #222 for theme variables and new  colors.
